cypress/integration/pages/articles/testsForAMPOnly.js

```diff
@@ -60,6 +63,98 @@ export const testsThatFollowSmokeTestConfigForAMPOnly = ({
         });
       });
     });
+
+    describe('Most Read Container', () => {
+      /* These cypress tests are needed as unit tests cannot be run on the jsdom.
+       * web workers (which run on amp pages) do not run on the virtual dom.
+       */
+      before(() => {
+        cy.getToggles(config[service].name);
+      });
+
+      const serviceVariant = variant === 'default' ? '' : `/${variant}`;
+
+      const mostReadPath = `/${config[service].name}/mostread${serviceVariant}.json`;
+
+      if (service !== 'scotland') {
+        it(`should show the correct number of items for ${service}\`s ${pageType}`, () => {
+          cy.request(mostReadPath).then(({ body: mostReadJson }) => {
+            const mostReadRecords = mostReadJson.totalRecords;
+            cy.fixture(`toggles/${config[service].name}.json`).then(toggles => {
+              const mostReadIsEnabled = path(['mostRead', 'enabled'], toggles);
+              cy.log(
+                `Most read container toggle enabled? ${mostReadIsEnabled}`,
+              );
+              if (
+                Cypress.env('APP_ENV') !== 'live' &&
+                mostReadIsEnabled &&
+                mostReadRecords > 5
```

Comment:
Should this be >= 5 (or > 4) as we want to run the test if there are 5 records? I think 5 is the minimum.

Reply:
Done 👍